dockstore-webservice/src/main/resources/migrations.1.11.0.xml

```diff
@@ -43,4 +43,29 @@
             </column>
         </createTable>
     </changeSet>
+    <changeSet author="ghogue (generated)" id="createAuthorTables">
+        <createTable tableName="author">
+            <column autoIncrement="true" name="id" type="BIGSERIAL">
+                <constraints primaryKey="true" primaryKeyName="author_pkey"/>
+            </column>
+            <column name="affiliation" type="VARCHAR(255)"/>
+            <column name="dbcreatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="dbupdatedate" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="email" type="VARCHAR(255)"/>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="orcid" type="VARCHAR(50)"/>
+            <column name="role" type="VARCHAR(255)"/>
+        </createTable>
+        <createTable tableName="version_author">
```

Comment:
Fair question

Reply:
> As it is now, I don't think you need a join table.

In @garyluu 's ORCID API storytime doc (see on Google Drive), the plan was to make Versions/Authors a ManyToMany relation.
If it were instead a OneToMany relation, then we wouldn't need a join table.

>Won't the current code create a new Author for every entry in the .dockstore.yml?

The current code has no authors in .dockstore.yml
Furthermore, the code in this PR doesn't actually create the authors from .dockstore.yml in the DB.
This is just setting up the (1) yml format for authors, and (2) the DB schema.

Reply:
> In @garyluu 's ORCID API storytime doc (see on Google Drive), the plan was to make Versions/Authors a ManyToMany relation.

Ah so I see.

> The current code has no authors in .dockstore.yml

By current code, I meant your PR, which was poor wording on my part.

But yes, looking at it again, you don't actually do anything with the Authors in the .dockstore.yml

I'm still concerned how you will avoid duplicating authors, or maybe that's the intention. But I guess we can cross that bridge further down the line and revisit.

Reply:
> I'm still concerned how you will avoid duplicating authors, or maybe that's the intention. But I guess we can cross that bridge further down the line and revisit.

Yeah, that was part of the hope here - to prompt that discussion since it's definitely not trivial.

In the current plan where it seems like we'd have to just "do our best" to avoid duplicates.
- Look for matching orcid
- Look for matching email
- Technically anyone could set anyone else as an author with inconsistent info
  + Do we just create a separate author entry if something doesn't match with the existing entry?

Definitely tricky.